build/dockerfiles/linux-libc.Dockerfile

```diff
@@ -120,6 +126,18 @@ RUN [[ $(uname -m) == "x86_64" ]] && NODE_ARCH=$(echo "console.log(process.arch)
     VSCODE_REMOTE_SERVER_PATH="$(pwd)/../vscode-reh-web-linux-${NODE_ARCH}" \
     /opt/app-root/src/retry.sh -v -t 3 -s 2 -- timeout -v 5m yarn smoketest-no-compile --web --headless --electronArgs="--disable-dev-shm-usage --use-gl=swiftshader"
 
+#########################################################
+#
+# Copy VS Code launcher to the container
+#
+#########################################################
+COPY launcher /checode-launcher
+WORKDIR /checode-launcher
+RUN yarn \
+    && mkdir /checode/launcher \
+    && cp -r out/src/*.js /checode/launcher \
+    && chgrp -R 0 /checode && chmod -R g+rwX /checode
```

Comment:
I guess the same changes are required for the downstream

Reply:
@vitaliy-guliy 
the current PR was merged
are you going to provide similar changes to the downstream?

Reply:
I think downstream already broken: 
- entrypoint in the downstream already contains your changes: https://github.com/redhat-developer/devspaces-images/blob/devspaces-3-rhel-8/devspaces-code/build/scripts/entrypoint-volume.sh#L50
- but the dockerfile is not adapted...

Reply:
I suppose the downstream was updated automatically
https://github.com/redhat-developer/devspaces-images/tree/devspaces-3-rhel-8/devspaces-code

Reply:
Did you check the corresponding dockerfile?
I believe the similar changes are required for the dockerfile...

Reply:
Done https://github.com/redhat-developer/devspaces-images/pull/491